src/dmap_import/pipeline.py

```diff
@@ -12,6 +14,8 @@ def start() -> None:
     print(f"using SQLAlchemy Version {sqlalchemy.__version__}")
     print(f"using alembic Version {alembic.__version__}")
 
+    alembic_upgrade_to_head()
+
     while True:
```

Comment:
question: I missed this before, but since this is a task which is expected to complete, should there be a `while True:` loop?

Reply:
I will drop the While Loop